### DISCUSSION TITLE
Inbound Messages duplicate

### DISCUSSION THREAD

**Discussion by francescodiperna** [OxalisCommunity/Oxalis-AS4#220] [General]
Good morning, we got 2 messages using oxalis that have the same content but in two different trasmission and different filename.
They arrived sequentially.
Is an error of the peppol network or an oxalis issue?

Kind regards
Di Perna Francesco

**Comment by dladlk** [comment]
Morning! Did you exclude the possibility that sender actually sent these 2 messages?

**Comment by francescodiperna** [comment]
> Morning! Did you exclude the possibility that sender actually sent these 2 messages?

Thanks for reply, It could be possible. I would exclude any other reasons.
Bye
